cmd/controller-manager/main.go

```diff
@@ -67,6 +69,11 @@ func init() {
 }
 
 func main() {
+
+	go func() {
+		http.ListenAndServe("localhost:6060", nil)
+	}()
```

Comment:
This is already covered in https://github.com/kubestellar/kubestellar/blob/v0.27.1/pkg/controller/run.go#L67-L75